app/assets/javascripts/controllers/report_data_controller.js

```diff
@@ -198,6 +200,15 @@
     this.initController(this.initObject);
   };
 
+  ReportDataController.prototype.gtlSetOneRowActive = function(item, event) {
+    this.gtlData.rows.map(row => { row.selected = false });
+    var selectedItem = _.find(this.gtlData.rows, {long_id: item.long_id});
+    selectedItem.selected = true;
+    this.$window.sendDataWithRx({rowSelect: selectedItem});
+    ManageIQ.gridChecks =  [];
+    ManageIQ.gridChecks.push(item.long_id);
```

Comment:
Ideally you should not be rewriting `gridCheks`, this will be replaced every time the user actually clicks on a checkbox, by `onItemSelect`. You probably can't use that as it has 0 support for having only 1 checkbox checked, but still, we should at least keep it consistent:

* don't set `gridChecks` to an empty array, `initObjects` already does that,
* keep `item.selected` and `item.checked` in sync,
* populate `gridChecks` from `checked` rows

What about:

```javascript
this.gtlData.rows.map(function(row) {
  row.checked = false;
  row.selected = false;
});

var selectedItem = _.find(this.gtlData.rows, {long_id: item.long_id});
selectedItem.checked = true;
selectedItem.selected = true;

this.$window.sendDataWithRx({rowSelect: selectedItem});

ManageIQ.gridChecks = this.gtlData.rows.filter(function(row) {
  return row.checked;
}).map(function(row) {
  return row.long_id;
}) || [];
```

Reply:
Since there are no checkboxes[Update Screenshot], I think it is ok as i is now.